taurus/entity/object/process_run.py

```diff
@@ -107,3 +109,17 @@ def template(self):
             return self.spec.template
         else:
             return None
+
+    @property
+    def _forward(self) -> Iterable[str]:
+        """List of forward links that should be skipped when doing a chronological traversal.
+
+        Usually, this is the same as the skip list.  However, the link direction between
```

Comment:
Referencing the documentation of `BaseEntity._forward` here, rather than duplicating it, might reduce the future cost of avoiding diverging documentation.